src/webots/gui/WbUpdatedDialog.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 1996-2021 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "WbUpdatedDialog.hpp"
+
+WbUpdatedDialog::WbUpdatedDialog(QWidget *parent) : QDialog(parent) {
+  setWindowTitle(tr("Welcome to Webots R2022a"));
+
+  QPixmap webotsLogo("images:webots.png");
+  QLabel *mImage = new QLabel(this);
```

Comment:
We should rename this variable to `image`, otherwise, it looks like a member variable which it isn't.
There are many similar cases below.